### DISCUSSION TITLE
Is Coral/TPU supported for acceleration of model/compute?

### DISCUSSION THREAD

**Discussion by typoworx-de** [mudler/LocalAI#603] [Q&A]
I'm having coral tpu device (USB) and I'm asking myself if this device could help speeding up compute for the models? Is this supported somehow?

Is this announcement related to this project?
> Introducing Coral: Our platform for development with local AI
https://developers.googleblog.com/2019/03/introducing-coral-our-platform-for.html

**Comment by imightbelosthere** [comment]
Hi,
I'm looking for the same type of information... did you ever found a way to get LocalAI working with the Coral TPU?

**Reply by Coolwrt** [reply]
HI, thank you `WELL`!

**Comment by MarvinSchenkel** [comment]
Also would love some insight in this. I am currently running Local AI on a NUC that does not have a way to add GPU acceleration

**Comment by zentaos** [comment]
Is there any progress on this? It would be awesome to boost AI without GPU

**Comment by BlackRockCity** [comment]
I'm interested in this too! Specifically, running Local AI in a Docker container and passing a USB Coral TPU (or perhaps M2) to the Docker container.

**Comment by pznamenskii** [comment]
same interest here. n150 mini-pc with m2 coral, would be awesome to use for LLMVision for HA
